src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -280,17 +270,17 @@ void generatePrecinctSummaryFiles(
       Map<String, Integer> numBallotsByPrecinct)
       throws IOException {
     Set<String> filenames = new HashSet<>();
-    for (String precinct : precinctRoundTallies.keySet()) {
-      String precinctFileString = getPrecinctFileString(precinct, filenames);
+    for (var entry : precinctRoundTallies.entrySet()) {
+      String precinctFileString = getPrecinctFileString(entry.getKey(), filenames);
```

Comment:
Nit: Might improve readability to factor out `String precinct = entry.getKey()` (and similarly for `entry.getValue()`) and inline `precinct` in each of these spots.

Reply:
Done.